src/scss/components/_contacts.scss

```diff
@@ -0,0 +1,10 @@
+.contacts {
+  padding: 50px 0 30px 0;
```

Comment:
You can use shorthand  `padding: 50px 0 30px;`

Reply:
done